media/vp8_ivf.ksy

```diff
@@ -7,6 +7,7 @@ meta:
   license: CC0-1.0
   ks-version: 0.7
```

Comment:
Given that you're adding the key `bit-endian: le` two lines below, which was added in the version 0.9 (see https://doc.kaitai.io/user_guide.html#bit-endian), please bump the declared minimal Kaitai Struct version needed to compile the spec:

```suggestion
  ks-version: 0.9
```

Reply:
I see. Is there a way to check those issues?

Reply:
> @generalmimon : I see. Is there a way to check those issues?

Not really. Sometimes we try to write the version when some feature was introduced to the [User Guide](https://doc.kaitai.io/user_guide.html) (search for `v0.` on the page to see all such occurences), but we still don't even have all features documented (i.e. particularly the ones added in the [0.9 release](https://kaitai.io/news/2020/10/16/kaitai-struct-v0.9-released.html)), so some are a bit esoteric.

But the `ks-version` key is not something to care about too much, because with every new release, we treat all older versions as EOL and we discourage to use them.